### PR TITLE
migrated es5 code to es6

### PR DESCRIPTION
**Pull request by adityaparab**
Addresses #11

### CODE REVIEW

File: lib/game_classes/Board.js [hunk 1]
```diff
@@ -1,78 +1,80 @@
...
 
-Board.prototype.inspect = function () {
-    var horizontalDivide = '|';
-    for (var i=0; i<this.lengthOfSide; i++) {
-        var line = '|';
-        for (var j=0; j<this.lengthOfSide; j++) {
-            line += this.tiles[i][j].getCode() + '|';
+    inspect () {
+        var horizontalDivide = '|';
+        for (var i = 0; i < this.lengthOfSide; i++) {
```
Comment (Asuza) [inline lib/game_classes/Board.js:15]: Please update all `var` to be `let`.

Comment (adityaparab) [reply lib/game_classes/Board.js:15]: @Asuza : Sure.. I mistakenly raised PR. Will make another commit in some time.

Comment (Asuza) [reply lib/game_classes/Board.js:15]: Oh, sorry. Just let me know when you're done with the PR. 😄

**Comment by Asuza** [comment]
@adityaparab, according to the [Hacktober rules,](https://hacktoberfest.digitalocean.com/#details) the pull requests must be created between during Oct 1 - Oct 31. So, you should cancel this PR entirely and create a new one for the issue on the 1st so it can count towards the t-shirt (if you're aiming for that, anyway).

If you want to complete the PR anyway, that's fine, too! Just wanted to warn you.

**Comment by adityaparab** [comment]
@Asuza : Thanks. Will close the PR and recreate it in said timeline

**Pull request closed without merging**